protocol/error.go

```diff
@@ -15,13 +15,33 @@ const (
 	ErrorMalformedClientMessage
 )
 
+const (
+	Passed ErrorCode = iota + 20
```

Comment:
Why is this `+ 20`?  (And why `+10` above?)  Why not just reuse Success?  Especially, since you're using the same map below.